internal/client/dep-cmd-flags.go

```diff
@@ -27,9 +27,6 @@ type DepCmdFlags struct {
 	flagMD  bool   // -MD (like -MF {def file})
 	flagMMD bool   // -MMD (mention only user header files, not system header files)
 	flagMP  bool   // -MP (add a phony target for each dependency other than the main file)
-
-	origO   string // if -MT not set, -o used as a target name (not resolved objOutFile, but as-is from cmdLine)
-	origCpp string // a first dependency is an input cpp file, but again, as-is, not resolved cppInFile
```

Comment:
See line 86

```go
if idx > 0 { // 0 is origCpp
```

Is it supposed to be like this, or did you forget to remove it?

Reply:
I've forgotten, fixed